examples/rust/components/sqldb-postgres-query/README.md

```diff
@@ -0,0 +1,151 @@
+# 🐘 SQLDB Postgres Example
+
+This folder contains a WebAssembly component that makes use of:
+
+- The [`wasmcloud:postgres` WIT contract][contract]
+- The [`sqldb-postgres-provider`][provider] Capability Provider
+
+[contract]: ./wit/deps/postgres/provider.wit
+[provider]: ../../../../crates/provider-sqldb-postgres
+
+## 📦 Dependencies
+
+- [`docker`][docker] for easily running instances of [`postgres`]
+- [`cargo`][cargo] (part of the Rust toolchain) for building this project
+- [`wash`][wash] for building and running the components and [wasmCloud][wasmcloud] hosts
+
+[docker]: https://docs.docker.com
+[postgres]: https://postgresql.org
+[cargo]: https://doc.rust-lang.org/cargo/
+
+## 👟 Quickstart
+
+As with all other examples, you can get started quickly by using whe [Wasmcloud SHell (`wash`)][wash].
+
+Since `wash` supports declarative deployments (powered by [Wasmcloud Application Deployment Manager (`wadm`)][wadm]), you can get started quickly using the `wadm.yaml` manifest in this folder:
+
+## Start a local Postgres cluster
+
+Before we can connect to a Postgres database cluster, we'll need to have one running. You can run one quickly with `docker`:
+
+```console
+docker run \
+    --rm \
+    -e POSTGRES_PASSWORD=postgres \
+    --name pg \
+    postgres:16.2-alpine
+```
+
+### Build this component
+
+```console
+wash build
+```
+
+This will create a folder called `build` which contains `sqldb_postgres_managed_s.wasm`.
+
+> [!NOTE]
+> If you're using a local build of the provider (using `file://...` in `wadm.yaml`) this is a good time to ensure you've built the [provider archive `par.gz`][par] for your provider.
+
+### Start a wasmCloud host
+
+```console
+wash up
+```
+
+> [!NOTE]
+> `wash up` will run as long as the host is running (you can cancel it with `Ctrl-C`)
+
+
+## Set up configuration for the provider
+
+Since configuration for Database clusters is usually sensitive information, we must pre-establish configuration for the provider using the [named configuration feature of wasmCloud][named-config].
+
+### Fully Managed connections
```

Comment:
I think this documentation needs to be updated as well

Reply:
You're right, done!